docs/src/demos/components/Menu/menu-section.demo.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Menu } from '@marigold/components';
+
+export const MenuWithSections = () => {
```

Comment:
I would like if there is also a Menu.Trigger around this :)

Reply:
**`Menu.Trigger`** has been added to the documentation demo .